multimodal/src/autogluon/multimodal/predictor.py

```diff
@@ -1496,11 +1501,12 @@ def _fit(
                 ".* in the `DataLoader` init to improve performance.*",
             )
             warnings.filterwarnings("ignore", "Checkpoint directory .* exists and is not empty.")
-            trainer.fit(
-                task,
-                datamodule=train_dm,
-                ckpt_path=ckpt_path if resume else None,  # this is to resume training that was broken accidentally
-            )
+            with set_torch_num_threads(num_cpus=num_cpus):
```

Comment:
Can we use lightning's trainer's `accelerator` and `devices` to setup cpu training?
https://lightning.ai/docs/pytorch/stable/common/trainer.html#accelerator
https://lightning.ai/docs/pytorch/stable/common/trainer.html#devices